diff -burNp nikto-2.1.1/nikto.conf nikto-2.1.1-dud/nikto.conf
--- nikto-2.1.1/nikto.conf	2010-02-02 20:28:54.000000000 +0100
+++ nikto-2.1.1-dud/nikto.conf	2010-02-07 00:28:22.201055867 +0100
@@ -22,10 +22,10 @@ USERAGENT=Mozilla/4.75 (Nikto/@VERSION) 
 #SKIPIDS=
 
 # if Nikto is having difficulty finding the 'plugins', set the full install path here
-# EXECDIR=/usr/local/nikto
+EXECDIR=/usr/share/nikto
 
 # The DTD
-NIKTODTD=docs/nikto.dtd
+NIKTODTD=/usr/share/doc/nikto/nikto.dtd
 
 # the default HTTP version to try... can/will be changed as necessary
 DEFAULTHTTPVER=1.0
@@ -68,6 +68,7 @@ CHECKMETHODS=HEAD GET
 
 # If you want to specify the location of any of the files, specify them here
 # EXECDIR=/opt/nikto
-# PLUGINDIR=/opt/nikto/plugins
-# TEMPLATEDIR=/opt/nikto/templates
-# DOCDIR=/opt/nikto/docs
+PLUGINDIR=/usr/share/nikto/plugins
+TEMPLATEDIR=/usr/share/nikto/templates
+DOCDIR=/usr/share/doc/nikto
+
